backend/src/controllers/metricsController.ts

```diff
@@ -41,20 +49,28 @@ const metricsController = {
       // Destructure req.body.promQLBody to get variables from the frontend.
       const { containerID, time } = req.query;
 
+      if (!time || !containerID) {
+        // error handler
+        return next();
+      }
+
+      // Parse time into Prometheus API friendly format
+      const promTime = (Date.parse(time.toString()) / 1000).toFixed(3);
```

Comment:
Is there a reason for invoking toString() and toFixed() here? I had thought toFixed() ultimately converts it to type string.

Reply:
Date.parse() requires the argument to be a string